internal/services/containerapps/helpers/container_apps.go

```diff
@@ -1565,6 +1573,12 @@ func ContainerVolumeMountSchemaComputed() *pluginsdk.Schema {
 					Computed:    true,
 					Description: "The path in the container at which to mount this volume.",
 				},
+
+				"sub_path": {
+					Type:        pluginsdk.TypeString,
+					Optional:    true,
```

Comment:
This should be computed only as it is used by a data source only.
```suggestion
					Computed:    true,
```